platform/features/table/res/templates/mct-table.html

```diff
@@ -1,22 +1,25 @@
-<div class="l-view-section scrolling"
-     ng-style="overrideRowPositioning ?
-         {'overflow': 'auto'} :
-         {'overflow': 'scroll'}"
-     >
-    <table class="filterable"
-           ng-style="overrideRowPositioning && {
+<div class="l-view-section scrolling" style="overflow: auto;">
```

Comment:
@charlesh88 There is some static markup here because `l-view-section` and `scrolling` both define different rules for overflow, and `hidden` wins. Not sure whether `scrolling` should have an !important, or whether I should just define a specific rule for this component?